config/index.js

```diff
@@ -92,6 +101,19 @@ function readRcConfig() {
 	return rc('suitest', {}, () => ({}));
 }
 
+/**
+ * Read josn config file provided by user.
+ * @param {string} configFile - path to config file
+ * @return {Object}
+ */
+function readUserConfig(configFile) {
+	// ignore config files when running unit tests
+	if (global._suitestTesting)
+		return {};
+
+	return JSON.parse(fs.readFileSync(configFile));
```

Comment:
One last thing. Make sure that a meaningful error is shown to user e.g. if there is no such file or if JSON is not valid. Process should end with error code in such case.

Reply:
Added description that it is cased by failing to process user config file. Plus original error message for more precise cause.